src/Components/SecondaryButton.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "../Button.css";
+
+const SecondaryButton = ({ children }) => {
```

Comment:
this can be combined with your primary button and take another prop for variant :)

Reply:
meaning you can set the class name based on the variant